vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -102,25 +102,25 @@ func (d *VolumeDriver) VolumesInRefMap() []string {
 // map hasn't been initialized, return 1 to prevent detach and remove.
 
 // Return the number of references for the given volume
-func (d *VolumeDriver) getRefCount(vol string) uint {
+func (d *VolumeDriver) getRefCount(vol string) (uint, error) {
 	if d.refCounts.GetInitSuccess() != true {
-		return 1
+		return 0, fmt.Errorf(plugin_utils.ErrorPluginInit)
 	}
-	return d.refCounts.GetCount(vol)
+	return d.refCounts.GetCount(vol), nil
 }
 
 // Increment the reference count for the given volume
-func (d *VolumeDriver) incrRefCount(vol string) uint {
+func (d *VolumeDriver) incrRefCount(vol string) (uint, error) {
 	if d.refCounts.GetInitSuccess() != true {
-		return 1
+		return 0, fmt.Errorf(plugin_utils.ErrorPluginInit)
 	}
-	return d.refCounts.Incr(vol)
+	return d.refCounts.Incr(vol), nil
 }
 
 // Decrement the reference count for the given volume
 func (d *VolumeDriver) decrRefCount(vol string) (uint, error) {
 	if d.refCounts.GetInitSuccess() != true {
-		return 1, nil
+		return 0, fmt.Errorf(plugin_utils.ErrorPluginInit)
 	}
 	return d.refCounts.Decr(vol)
```

Comment:
I see. But that means the existing refcount.Incr() and refcount.Decr() are not well designed. A good API design should make the same group of functions/methods consistent. Obviously we are violating this which has caused this confusion. Is it possible to improve these 2 functions to keep them consistent?